spark-indexer/src/main/java/org/uniprot/store/spark/indexer/suggest/mapper/document/GOToSuggestDocument.java

```diff
@@ -34,9 +34,11 @@
     }
 
     private SuggestDocument buildSuggestDocument(GeneOntologyEntry goTerm) {
+        String idOnly = goTerm.getId().substring(3);
```

Comment:
The code is fragile here. We are using magic number 3

Reply:
This is an ID field that will always have a prefix "GO:"